_pages/policies/travel-and-leave-policies/first-time-travel-get-in-concur.md

```diff
@@ -5,7 +5,7 @@ title: Get access to Concur
 [TTS Travel Guide Table of Contents]({{site.baseurl}}/travel-guide-table-of-contents)
 [Next to Travel Card]({{site.baseurl}}/first-time-travel-travel-card)
 
-1. To access [Concur](http://travel.gsa.gov/), you’ll first need to get a **“travel vendor ID”**. The easiest way to do this is to email kc-travel.finance@gsa.gov and request your vendor ID, and that payroll banking information can be used to generate this.
+1. To access [Concur](http://travel.gsa.gov/), you’ll first need to get a **“travel vendor ID”**. The easiest way to do this is to email [kc-travel.finance@gsa.gov](mailto:kc-travel.finance@gsa.gov). Your email should request your vendor ID and state that payroll banking information can be used to generate the ID.  We've drafted a template for you - just [click](mailto:kc-travel.finance@gsa.gov?subject=Request%20for%20Travel%20Vendor%20ID&body=Hi.%20I%20will%20be%20traveling%20for%20work%20and%20will%20need%20a%20travel%20vendor%20ID.%20Can%20you%20please%20assign%20one%20to%20me?%20My%20payroll%20banking%20information%20can%20be%20used%20to%20generate%20this%20ID.), personalize, and send.
```

Comment:
I worry that "click" is not quite predictable enough anchor text, from an accessibility standpoint. How about hyperlinking `template` or `email template`?

Reply:
Or perhaps change the text to "We've drafted an email template for you here." and hyperlink "email template for you here".

Reply:
Thanks y'all! Good feedback, I'll update that shortly.